ACD-618 (#30)
Actindo-AG/Documentation: Channels/UserInterface/04a_OfferImport.md

[!!Import offers](../Operation/02_ImportOffers.md)

# Offer import

*Omni-Channel > Offer import > Tab OFFER IMPORT*

![Offer import](../../Assets/Screenshots/Channels/OfferImport/OfferImport/OfferImport.png "[Offer import]")

**Importable products**

- [DOWNLOAD IMPORTABLE PRODUCTS]   
  Click this button to download all importable offers from the marketplace. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.

- [IMPORT ALL LISTED PRODUCTS TO CHANNELS]   

>   Click this button to import all importable offers from your marketplace at once to the *Omni-Channel* module. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.

Reviewer: ```suggestion
  Click this button to import all importable offers from your marketplace to the *Omni-Channel* module at once. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.
```

- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   

>   Click this button to display the search bar and search for a scheduled offer upload. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.

Reviewer: ```suggestion
  Click this button to display the search bar and search for an importable product. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.
```

- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   

>   Click this button to update the list of scheduled offer uploads. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.

Reviewer: ```suggestion
  Click this button to update the list of importable products. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.
```

      - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish    
        Click this entry to publish the view. This menu entry is only displayed if a view has been selected and unpublished.

      - ![Private view](../../Assets/Icons/Unpublish.png "[Unpublish]") private view    
        Click this entry to unpublish the view. This menu entry is only displayed if a view has been selected and published.

      - ![Save](../../Assets/Icons/Save.png "[Save]") save    
        Click this entry to save the current view settings in the selected view. This menu entry is only displayed if a view has been selected.

        > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.

      - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete    
        Click this entry to delete the selected view. A confirmation window is displayed to confirm the deletion. This menu entry is only displayed if a view has been selected.

> - *Select Connection*    

Reviewer: ```suggestion
- *Select connection*    
```

  Click the drop-down list to select a connection. All supported connections are displayed in the list. You must select a connection to display any offer imports in the list.

- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.

- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active. This button is only displayed if a connection has been selected in the *Select connection* drop-down list.

> 

Reviewer: Im Screenshot sieht man kein Filter... oder sieht man ein Filter, wenn es Einträge auf der Liste gibt?

- [x]     

>   Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all offer imports in the list are selected.

Reviewer: offer imports oder importable products? (vgl. oben, Beschreiben Search- und Refresh-Buttons)

- [IMPORT PRODUCT(S) TO CHANNELS]   
  Click this button to import the selected offer(s) to the *Omni-Channel* module. This button is only displayed if a connection has been selected in the *Select connection* drop-down list and if the checkbox of at least one offer is selected.

- [DETAILS]   
  Click this button to display the *Detail of log message "Log message title"* view and check further details of the error. This button is only displayed if a connection has been selected in the *Select connection* drop-down list and if the checkbox of a single offer with an error is selected.

> The list displays all offer imports. Depending on the settings, the displayed columns may vary. All fields are read-only. If no offer imports are available, the *Please select a connection to import products from* notice is displayed. The available columns differ depending on the selected connection in the *Select Connection* drop-down list. Therefore, only those columns that are available for all connections are described in the following.

Reviewer: s.o.

> The list displays all offer imports. Depending on the settings, the displayed columns may vary. All fields are read-only. If no offer imports are available, the *Please select a connection to import products from* notice is displayed. The available columns differ depending on the selected connection in the *Select Connection* drop-down list. Therefore, only those columns that are available for all connections are described in the following.

Reviewer: ```suggestion
The list displays all offer imports. Depending on the settings, the displayed columns may vary. All fields are read-only. If no offer imports are available, the *Please select a connection to import products from* notice is displayed. The available columns differ depending on the selected connection in the *Select connection* drop-down list. Therefore, only those columns that are available for all connections are described in the following.
```
"offer imports" (s.o.)
*Please select a connection to import products from* notice -> kommt be mir nicht vor, wieso?

- *Error message*   

>   Error log message if an error has occurred. Select the checkbox of the corresponding row and click the [DETAILS] button in the editing toolbar to display further details to the error. If no error has occurred, the field is empty.

Reviewer: If no error has occurred, the field is empty. --> Welches Feld? Ich kann das nicht überprüfen, denn es gibt keine Fehler bei mir, aber wieso ist das möglich? Wenn es keinen Fehler gibt, sollte es auch keine Error message geben, oder?
